src/domain_replacements.py

```diff
@@ -0,0 +1,9 @@
+# domain_replacements.py
+# -*- coding: utf-8 -*-
+domain_replacements = {
+    "https://twitter.com/": "https://www.fxtwitter.com/",
+    "https://www.instagram.com/p/": "https://www.ddinstagram.com/p/",
+    "https://www.instagram.com/reel/": "https://www.ddinstagram.com/reel/",
+    "https://www.twitter.com/": "https://www.fxtwitter.com/",
+    "https://x.com/": "https://www.fxtwitter.com/",
+}
```

Comment:
Here we use number of aliases to later replace them with a single value

Another way to do so is composing a dict of mask-replacement pairs:

```suggestion
url_replacements_dict = {
    r"https://(www\.)?(twitter|x)\.com/": "https://www.fxtwitter.com/",
    r"https://(www\.)?instagram\.com/(p|reel)/": "https://www.ddinstagram.com/\2/",
}

def replace_url():
    for mask, replacement in url_replacements_dict.items():
        if re.compile(mask).match(url):
            url = re.sub(mask, replacement, url)
            return url
    return url
```